src/WebpGenerator.php

```diff
@@ -23,6 +23,10 @@ public function generate(string $url, string $mimeType): string
             return $url;
         }
 
+        if (strpos($_SERVER['HTTP_ACCEPT'], 'image/webp') === false) {
```

Comment:
Since the `HTTP_ACCEPT` header isn't required we probably also want to check whether it is set, before matching.